cloudformation/Application.yml

```
@@ -0,0 +1,397 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: EC2 RDS ALB S3 create
+
+############################################
+#         Metadata & Parameter
+############################################
+Metadata: #入力パラメータのグループ化と順序の指定
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      -
+        Label:
+          default: Can be created as a separate resource by renaming the environment
+        Parameters: 
+          - EnvironmentName

```

```suggestion
Metadata: #入力パラメータのグループ化と順序の指定
  AWS::CloudFormation::Interface:
    ParameterGroups:
      - Label:
          default: Can be created as a separate resource by renaming the environment
        Parameters: 
          - EnvironmentName

```
yamlの配列の最初を改行するのはあまりやらないですね。細かいルールは現場によりますが！
少なくとも、他の配列（IAMポリシー）は改行されていないので改行なしで統一させていていただいた方がよいですね！

---

改行をなくしました。